src/components/Profile/components/PersonalInfoList/index.jsx

```diff
@@ -375,6 +383,123 @@ const PersonalInfoList = ({ myProf, profile, isOnline, createSnackbar }) => {
                     {showMailCombo ? <VisibilityOffIcon /> : <VisibilityIcon />}
                   </IconButton>
                 </Grid>
+                <Button
+                  variant="contained"
+                  color="primary"
+                  onClick={() => setIsJoinDialogOpen(true)}
+                >
+                  Instructions
+                </Button>
+                <GordonDialogBox
+                  open={isJoinDialogOpen}
+                  title={`Mailbox Instructions`}
+                  closeButtonClicked={onClose}
+                  maxWidth="md"
+                >
+                  <Grid container>
+                    <Typography sx={{ fontSize: '0.8rem' }}>
+                      <Link
+                        className="disable_select"
+                        style={{
+                          borderBottom: '1px solid currentColor',
+                          textDecoration: 'none',
+                          color: gordonColors.primary.blueShades.A700,
+                          fontSize: '1.1rem',
+                          fontWeight: 'bold',
+                        }}
```

Comment:
```suggestion
                        style={{
                          borderBottom: '1px solid currentColor',
                          textDecoration: 'none',
                          color: gordonColors.primary.blueShades.A700,
                          fontSize: '1.1rem',
                          fontWeight: 'bold',
                        }}
```

Is there specific reason you don't have your styling inside of a `css` file?